client/my-sites/stats/all-time-highlights-section/index.tsx

```diff
@@ -1,9 +1,33 @@
 import { Card } from '@automattic/components';
 import { Icon, people, postContent, starEmpty, commentContent } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
```

Comment:
Could we avoid using moment? Its inclusion has big performance implications that we'll need to address in the future (by removing it from our code base).

Reply:
Resolved in [a0e623c](https://github.com/Automattic/wp-calypso/pull/70001/commits/a0e623c48562c98f0fe8cd36c7bf8a0b629797f9).